### PR TITLE
Clarify on the crossorigin attribute

### LINKED ISSUE #69: Explain further the crossorigin attribute usage
So in the ember addon I have explained a little further about the `crossorigin` attribute usage:
https://github.com/jonathanKingston/ember-cli-sri

Something like:
## Crossorigin attribute

When the request doesn't match Same Origin Policy the [crossorigin attribute](https://html.spec.whatwg.org/multipage/infrastructure.html#cors-settings-attribute) **MUST** be present for the integrity of the file to be checked.
With an integrity set on an external origin and a missing crossorigin then the browser will choose to 'fail-open' which means it will load the resource as if the integrity attribute was not set.

Values:
- **anonymous** - A cross-origin request (i.e., with Origin: HTTP header) is performed. But no credential is sent (i.e., no cookie, no X.509 certificate, and no HTTP Basic authentication is sent). If the server does not give credentials to the origin site (by not setting the Access-Control-Allow-Origin: HTTP header), the resource will be tainted and its usage restricted.
- **use-credentials** - A cross-origin request (i.e., with Origin: HTTP header) performed with credential is sent (i.e., a cookie, a certificate, and HTTP Basic authentication is performed). If the server does not give credentials to the origin site (through Access-Control-Allow-Credentials: HTTP header), the resource will be tainted and its usage restricted.

### PR DESCRIPTION
**Pull request by jens1o**
fixes #69

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-    
+
     <!--  Facebook -->
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://www.srihash.org/">
     <meta property="og:title" content="SRI Hash Generator">
     <meta property="og:description" content="Generate Subresource Integrity (SRI) attributes, easily and fast">
     <meta property="og:image" content="">
-    
+
```
Comment (mozfreddyb) [inline index.html:35]: ...thanks for fixing the newlines along the way.

File: index.html [hunk 2]
```diff
@@ -84,6 +84,20 @@ <h2>How can I generate Integrity hashes?</h2>
...
+      <h2>Why do I need to include <code class="inline">crossorigin="anonymous"</code>?</h2>
+      <p>
+        When the request is not on the same origin the <code class="inline">crossorigin</code> attribute must be present to check the integrity of the
+        file.<br>
+        Without a <code class="inline">crossorigin</code> attribute, the browser will choose to 'fail-open' which
+        means it will load the resource as if the integrity attribute was not set, effectively losing all the security SRI brings in the first place.<br><br>
+        <code class="inline">crossorigin="anonymous"</code> results that no credentials are sent to the cross-origin site hosting the content. However,
+        it will send an <code class="inline">Origin</code> HTTP header. If the server denies including the resource (by not setting the <code class="inline">Access-Control-Allow-Origin</code> HTTP header),
+        the resource will not be used by the browser.<br>
+        You can find more information on <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin" target="_blank" rel="noopener noreferrer">MDN.</a>
```
Comment (mozfreddyb) [inline index.html:96]: Please spell MDN out.

**Review by mozfreddyb — CHANGES_REQUESTED** [review]
Almost done. Can you do a test deployment under your own github pagesf for me to test?

**Comment by jens1o** [comment]
@mozfreddyb Are you sure you want to test the simple addition of HTML and not rather the syntax highlighting PR?

**Comment by mozfreddyb** [comment]
> @mozfreddyb Are you sure you want to test the simple addition of HTML and not rather the syntax highlighting PR?

Oops. Thank you!

**Pull request merged**